types/errors.go

```diff
@@ -246,19 +247,22 @@ func (err *sdkError) Code() CodeType {
 
 // Implements ABCIError.
 func (err *sdkError) ABCILog() string {
-	cdc := codec.New()
 	errMsg := err.cmnError.Error()
 	jsonErr := humanReadableError{
 		Codespace: err.codespace,
 		Code:      err.code,
 		Message:   errMsg,
 	}
-	bz, er := cdc.MarshalJSON(jsonErr)
-	if er != nil {
-		panic(er)
+
+	var buff bytes.Buffer
+	enc := json.NewEncoder(&buff)
+	enc.SetEscapeHTML(false)
+
+	if err := enc.Encode(jsonErr); err != nil {
+		panic(errors.Wrap(err, "failed to encode ABCI error log"))
 	}
-	stringifiedJSON := string(bz)
-	return stringifiedJSON
+
+	return strings.TrimSpace(buff.String())
```

Comment:
the @cosmos/cosmos-ui was hoping for something more like:
```
{
  "codespace": "sdk",
  "code": 10,
  "message": "You are trying to send more stake than you own."
}
```
(from https://github.com/cosmos/cosmos-sdk/issues/3601)

Reply:
is this the expected response?

Reply:
Thanks for following up @jbibla! Are you saying you wish it to be pretty JSON and indented? It already is JSON.